app/utils/utils.go

```diff
@@ -72,3 +74,18 @@ func ExtractQueueAttributes(u url.Values) map[string]string {
 	}
 	return attr
 }
+
+func CreateErrorResponseV1(errKey string, isSqs bool) (int, interfaces.AbstractResponseBody) {
```

Comment:
I like to use type alias like `AwsProtocol`.
e.g.
```
type ErrorType int

const (
	SqsError  ErrorType = iota
	SnsError ErrorType = iota
)
```

Reply:
Let's hold there - there's only 2 options at the moment.  If we need more than that, or it gets too confusing, we can swap to another pattern like the `AwsProtocol`.  How does that sound?

Reply:
It's OK!